generate/generate.go

```diff
@@ -238,6 +238,16 @@ func New(os string) (generator Generator, err error) {
 			Seccomp: seccomp.DefaultProfile(&config),
 		}
 	}
+	if os == "freebsd" {
+		config.Mounts = []rspec.Mount{
```

Comment:
For containerd, I also added fdescfs by default.  Not sure whether you'd want that here (or if maybe it should be removed from containerd's default).

Reply:
Good point, I added fdescfs as a default mount.